src/test/ui/did_you_mean/issue-38147-2.nll.stderr

```diff
@@ -2,7 +2,7 @@ error[E0596]: cannot borrow `*self.s` as mutable, as it is behind a `&` referenc
   --> $DIR/issue-38147-2.rs:17:9
    |
 LL |     s: &'a String
-   |     ------------- use `&'a mut String` here to make mutable
+   |     ------------- help: consider changing this to be mutable: `&'a mut String`
```

Comment:
The span starts too early: Can you make sure it only covers `&'a String`, as the replacement doesn't contain the `s: ` part?

Reply:
Fixed!